test/images/node-perf/tf-wide-deep/BASEIMAGE

```diff
@@ -1 +1,2 @@
 amd64=python:3.6-slim-stretch
+arm64=arm64v8/python:3.6-slim-stretch
```

Comment:
do we want to add other arches too?

Reply:
I have tried it, and I have not found the required tensorflow package on other non-amd64 platforms. Such as, I tried to install tensorflow or tensorflow-ppc64le on ppc64le platform, but  failed.

Reply:
for ppc64le we do have ppa published but its not that straight forward. I will work with the concerned team in IBM and push that PR later.